python/GafferOSL/FocalBlur.gfr

```diff
@@ -0,0 +1,889 @@
+__children["FocalBlur"]["OSLCode1"]["__uiPosition"].setValue( imath.V2f( 801.823242, 12.3714523 ) )
+__children["FocalBlur"]["Shuffle8"]["in"].setInput( __children["FocalBlur"]["Dot17"]["out"] )
+__children["FocalBlur"]["Shuffle8"]["shuffles"]["shuffle0"]["source"].setValue( 'A' )
+__children["FocalBlur"]["Shuffle8"]["shuffles"]["shuffle0"]["destination"].setInput( __children["FocalBlur"]["depthChannel"] )
+__children["FocalBlur"]["Shuffle8"]["__uiPosition"].setValue( imath.V2f( 849.973633, 66.9294434 ) )
+__children["FocalBlur"]["Grade14"]["in"].setInput( __children["FocalBlur"]["Shuffle8"]["out"] )
+__children["FocalBlur"]["Grade14"]["channels"].setInput( __children["FocalBlur"]["depthChannel"] )
+__children["FocalBlur"]["Grade14"]["blackPoint"].setValue( imath.Color4f( 1, 0, 0, 0 ) )
+__children["FocalBlur"]["Grade14"]["whitePoint"].setValue( imath.Color4f( 0, 1, 1, 1 ) )
+__children["FocalBlur"]["Grade14"]["multiply"]["r"].setInput( __children["FocalBlur"]["backgroundZValue"] )
+__children["FocalBlur"]["Grade14"]["blackClamp"].setValue( False )
+__children["FocalBlur"]["Grade14"]["__uiPosition"].setValue( imath.V2f( 849.973633, 61.2653847 ) )
+__children["FocalBlur"]["Dot12"]["in"].setInput( __children["FocalBlur"]["BoxIn"]["out"] )
+__children["FocalBlur"]["Dot12"]["__uiPosition"].setValue( imath.V2f( 839.474304, 92.0897293 ) )
+__children["FocalBlur"]["Merge14"]["in"][0].setInput( __children["FocalBlur"]["DeleteChannels"]["out"] )
+__children["FocalBlur"]["Merge14"]["in"][1].setInput( __children["FocalBlur"]["Dot17"]["out"] )
+__children["FocalBlur"]["Merge14"]["operation"].setValue( 9 )
+__children["FocalBlur"]["Merge14"]["__uiPosition"].setValue( imath.V2f( 860.447693, 47.4372597 ) )
+__children["FocalBlur"]["DeleteChannels"]["in"].setInput( __children["FocalBlur"]["Grade14"]["out"] )
+__children["FocalBlur"]["DeleteChannels"]["mode"].setValue( 1 )
+__children["FocalBlur"]["DeleteChannels"]["channels"].setInput( __children["FocalBlur"]["depthChannel"] )
+__children["FocalBlur"]["DeleteChannels"]["__uiPosition"].setValue( imath.V2f( 849.973633, 55.6013222 ) )
+__children["FocalBlur"]["Backdrop"]["title"].setValue( 'Fix Background Depth Value ( Arnold )' )
+__children["FocalBlur"]["Backdrop"]["__uiPosition"].setValue( imath.V2f( 850.59137, 55.5018997 ) )
+__children["FocalBlur"]["Backdrop"]["__uiBound"].setValue( imath.Box2f( imath.V2f( -14.8830109, -20.1147346 ), imath.V2f( 38.0354614, 29.9420242 ) ) )
+__children["FocalBlur"]["Dot14"]["in"].setInput( __children["FocalBlur"]["SetBlurRadius"]["out"] )
+__children["FocalBlur"]["Dot14"]["__uiPosition"].setValue( imath.V2f( 940.699768, 5.03942108 ) )
+__children["FocalBlur"]["focusDistance"].setValue( 70.0 )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["focusDistance"], 'noduleLayout:section', 'left' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["focusDistance"], 'description', 'The Z value that is perfectly in focus.' )
```

Comment:
We should get rid of the references to "Z" here and in `computedFocusDistance`'s description.